templates/add.jinja2

```diff
@@ -12,29 +12,33 @@
 <body>
```

Comment:
Why are you not inheriting the general structure of your page from a base jinja template?  You've got a lot of repeated HTML here that will become a nightmare to maintain over time.